metaproc-connector-demo/config/variables.yaml

```diff
@@ -3,4 +3,5 @@ Variables:
     Url: http://localhost:8080
     Username: admin
     Password: 1234
-    TokenExpiredDuration: 60 # Minute
+    # Token expiration in minute
+    TokenExpiredDuration: 60
```

Comment:
is 60 minutes a default we know from meta-proc? or do we expect all customers to customize this? maybe @thomaszehetner has some insights here?

Reply:
I don't know exactly what type of token this API uses. So I made it editable for manual expire logic.

Reply:
feedback from meta:proc: default token expiration time is 60 minutes

Reply:
Thanks a lot for your feedback @thomaszehetner.